fastplotlib/graphics/image.py

```diff
@@ -76,18 +72,14 @@ def __init__(
             pygfx.ImageBasicMaterial(clim=(vmin, vmax), map=get_cmap_texture(cmap))
         )
 
-    @property
-    def indices(self) -> Any:
-        pass
-
     @property
     def features(self) -> List[str]:
-        pass
+        return ["cmap", "data"]
```

Comment:
This won't be necessary after the new #78  :smile: (keep it in for now for testing w.r.t. current `master` branch)